src/scf-release/src/acceptance-tests-brain/test-resources/nfs-server/README.md

```diff
@@ -0,0 +1,31 @@
+# NFS server for Docker
+Docker nfs-server based on CentOS. To use, run the image as follows:
+
+```
+# Enable NFS modules
+sudo modprobe nfs
+sudo modprobe nfsd
+
+docker run -d --name nfs \
+    -v "/home/toaster/tools/nfs_share:/exports/foo" \
+    -p 111:111/tcp \
+    -p 111:111/udp \
+    -p 662:662/udp \
+    -p 662:662/tcp \
+    -p 875:875/udp \
+    -p 875:875/tcp \
+    -p 2049:2049/udp \
+    -p 2049:2049/tcp \
+    -p 32769:32769/udp \
+    -p 32803:32803/tcp \
+    -p 892:892/udp \
+    -p 892:892/tcp \
+    --privileged \
+    viovanov/nfs-server /exports/foo
```

Comment:
Is this still the image name? I thought this was moved to live under the `splatform` org.

Reply:
When I copied the source files I did not make changes to them.
Yes, the image we use is at splatform.
I can change the README.

Reply:
Rebased. Comment address with commit 8ff62ff711d40a3832130aa69b8f6c4cc6e17aed

Reply:
Can you update the README at the root of this repo as well?
https://github.com/SUSE/scf/blame/aku-b5kQaco7-nfs-test/README.md#L766

Reply:
Yes. Done with commit 7dd73eb5f92be7c1fa204c600bcdd6377a7eca9e